src/main/java/helper/DataHelper.java

```diff
@@ -0,0 +1,20 @@
+package helper;
+
+import com.github.javafaker.Faker;
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class DataHelper {
+    private static final Faker faker = new Faker();
+
+    public static String getRandomEmail() {
+        return faker.letterify("??????????@gmail.com");
+    }
+
+    public static String getRandomPassword(int number) {
+        return RandomStringUtils.randomAlphanumeric(number);
+    }
+
+    public static String getRandomPID(int number) {
```

Comment:
please rename this method to reuse in others cases, not just pid

Reply:
fixed